README.md

```diff
@@ -2,6 +2,18 @@
 
 A java integration library for AeroGear Digger
 
+## Project Info
+
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
+| Documentation:  | tbd  |
```

Comment:
no, that's stupid, doing self reference.

once we have somehting real, on aerogear.org/digger, we can put stuff in there